addons/account_edi/views/account_move_views.xml

```diff
@@ -51,6 +51,7 @@
                                 <field name="edi_format_name"/>
                                 <field name="state"/>
                                 <field name="error" invisible="1"/>
+                                <button name="action_export_xml" type="object" class="oe_link oe_inline" string="Download" groups="base.group_no_one" attrs="{'invisible': [('error', '=', False)]}"/>
```

Comment:
Need to be overridden in `account_edi_extended` to add the blocking level inside:
`['|', ('error', '=', False), ('blocking_level', '=', 'warning')]`

Reply:
Also, can you indent it plz? This line is too long :p

Reply:
Indented, but please explain the requested change in `account_edi_extended`

Reply:
done